exercises/simple-cipher/canonical-data.json

```diff
@@ -87,7 +88,7 @@
           "expected": "qayaeaagaciai"
         },
         {
-          "description": "Can wrap on encode", 
+          "description": "Can wrap on encode",
```

Comment:
Please try to limit your changes to the scope of the PR.  This instance is minor.

Reply:
This is most likely automatic because of an IDE.

Reply:
My Emacs configuration automatically removes trailing whitespace on save. And I had prettier enabled also automatically on save. Shouldn't have committed those... All formatting changes not related to this PR now removed.